python/openai/openai/src/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
agree we should come up with a general scheme for layout - can be later - but for reference in other project have been following:

<library/package><language><package-name>

So maybe something like

```openai_frontend/python/openai_frontend``
``openai_frontend/rust/...```
``openai_frontend/python/tests```

Still has the double "openai_frontend" - but one is the logical feature / library and one is the language specific directory structure. 
 for c++ it might be

``openai_frontend/c++/src/```

As the library naming isn't direcctory dependent .

Reply:
I would rather `<language>/<library/package>/<language-specific-package-name>`, which is the common practice for repos that have support for different languages, and much easier to build / package library for corresponding language.

Reply:
@GuanLuo - do you have a reference we can use? - we can add to help guide

Reply:
You don't need to follow the same file structure within language barrier, but the language barrier should make sure the files within will be self-contained.
https://github.com/NVIDIA/cutlass
https://github.com/rapidsai/ucxx/tree/branch-0.39
https://github.com/tensorflow/tensorflow/tree/master/tensorflow

Reply:
Updated to the following for now based on the thread:
- `python/openai/tests`
- `python/openai/openai_frontend`
- Removed `src`

ref: https://github.com/triton-inference-server/server/pull/7561/commits/886ee7d212a8f6774d6dfefdeffeb328f32fad24

Happy to rename folders or re-structure later after addressing bigger design feedback